GVRf/Framework/src/org/gearvrf/GVRLight.java

```diff
@@ -29,8 +29,49 @@ public GVRLight(GVRContext gvrContext) {
         setDiffuseIntensity(1.0f, 1.0f, 1.0f, 1.0f);
         setSpecularIntensity(1.0f, 1.0f, 1.0f, 1.0f);
         isEnabled = true;
+        name="";       
```

Comment:
Please use the same spacing format as the rest of the code. I found this is quite similar to what we use in this project: http://www.javaranch.com/styleLong.jsp#spacing